Source/WebCore/platform/graphics/texmap/TextureMapperFPSCounter.cpp

```diff
@@ -23,6 +23,7 @@
 
 #include "TextureMapperFPSCounter.h"
 
+#include "Page.h"
```

Comment:
This is a layering violation, platrform layer should not use classes from WebCore. I think we could probably move the FPS counter to WebKit layer, since it's always used from the compositor, no?